src/main/resources/data/jobProfiles/defaultJobProfiles.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "19c33d10-99d2-11ea-bb37-0242ac130002",
+  "name": "Default job profile",
+  "destination": "fileSystem",
+  "description": "Default job profile",
+  "userInfo": {
+    "firstName": "System",
+    "lastName": "Process",
+    "userName": "system_process"
+  },
+  "mappingProfileId": "25d81cbe-9686-11ea-bb37-0242ac130002",
+  "metadata": {
```

Comment:
we can remove the metadata, RMB should add it since we are using the API

Reply:
sure,will remove it